packages/sdks/typescript/src/index.ts

```diff
@@ -156,7 +154,39 @@ class Latitude {
       })
     }
 
-    return (await response.json()) as DocumentVersion & { config: Config }
+    return (await response.json()) as Prompt
+  }
+
+  async getOrCreatePrompt(
+    path: string,
+    { projectId, versionUuid, prompt }: GetOrCreatePromptOptions = {},
+  ) {
+    projectId = projectId ?? this.options.projectId
+    if (!projectId) {
+      throw new Error('Project ID is required')
+    }
+    versionUuid = versionUuid ?? this.options.versionUuid
+
+    const response = await makeRequest({
+      method: 'POST',
+      handler: HandlerType.GetOrCreateDocument,
+      params: { projectId, versionUuid },
+      options: this.options,
+      body: { path, prompt },
+    })
+
+    if (!response.ok) {
+      const json = (await response.json()) as ApiErrorJsonResponse
+      json.errorCode
```

Comment:
iepa

Reply:
oops copied from `get` without thinking... will change it there to!